core/src/main/scala/com/databricks/labs/remorph/generators/sql/LogicalPlanGenerator.scala

```diff
@@ -33,6 +33,7 @@ class LogicalPlanGenerator(val expr: ExpressionGenerator, val explicitDistinct:
     case i: ir.InsertIntoTable => insert(ctx, i)
     case ir.DeleteFromTable(target, None, where, None, None) => delete(ctx, target, where)
     case ir.NoopNode => ""
+    case unresolved: ir.UnresolvedCommand => s"--${unresolved.inputText}"
```

Comment:
```suggestion
    case ir.UnresolvedCommand(inputText) => s"-- ${inputText}"
```
be a bit more scala idiomatic

Reply:
done